akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -428,6 +442,9 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `audio/xm`          = aud("xm", NotCompressible, "xm")
   val `audio/webm`        = aud("webm", NotCompressible)
 
+  val `font/woff`  = font("woff", NotCompressible, "woff")
```

Comment:
Perhaps link to the RFC in a comment here?

Reply:
Added